Scripts/Python/ki/__init__.py

```diff
@@ -5520,6 +5527,15 @@ def ProcessNotifyMicro(self, control, event):
             ctrlID = control.getTagID()
             if ctrlID == kGUI.ChatEditboxID:
                 self.Autocomplete(control)
+        # Up or Down key to scroll in the chat history
+        elif event == kMessageHistoryUp:
+            ctrlID = control.getTagID()
+            if ctrlID == kGUI.ChatEditboxID:
+                self.MessageHistory(control, "up")
+        elif event == kMessageHistoryDown:
+            ctrlID = control.getTagID()
+            if ctrlID == kGUI.ChatEditboxID:
+                self.MessageHistory(control, "down")
```

Comment:
I thought the KI already did this?

Reply:
Only in ProcessNotifyMini(), not in ProcessNotifyMicro().  This won't change the world, but it does fix a needless inconsistency.